mlir_graphblas/src/include/GraphBLAS/GraphBLASUtils.h

```diff
@@ -0,0 +1,23 @@
+#ifndef GRAPHBLAS_GRAPHBLASUTILS_H
+#define GRAPHBLAS_GRAPHBLASUTILS_H
+
+#include <string>
+#include "mlir/IR/BuiltinOps.h"
+#include "mlir/Dialect/StandardOps/IR/Ops.h"
+#include "mlir/Dialect/Tensor/IR/Tensor.h"
+
+
```

Comment:
Would it be easier if we used `using namespace mlir` here?

That might let us get rid of the "mlir::" prefixes everywhere.

Reply:
Old C++ habit.  Maybe this is not true anymore, but I thought that putting `using` statements into headers was frowned upon because it implicitly opts every .cpp file including that header (including external projects) into also automatically searching that namespace.

Reply:
I think you're right. That might be a bad idea. Maybe it'd be a better idea to do something like [this](https://github.com/llvm/llvm-project/blob/main/mlir/include/mlir/Dialect/SparseTensor/IR/SparseTensor.h#L27):
```
namespace mlir {
   // insert declarations here
} // namespace mlir
```

Reply:
That also has the effect of putting these new functions into the `mlir` namespace as well, which we also probably don't want to do since these functions aren't part of MLIR.

Reply:
Going to leave this as-is.